Color.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
```

Comment:
you do not create AreaCalculator interface

Reply:
done